app/client/render.js

```diff
@@ -18,6 +19,8 @@ export function preRenderApplication(store) {
   };
 
   global.qtLoadedFromShell && renderComponent(Footer, "footer", store, hydrate);
+  renderComponent(AppLogo, "app-logo", store, hydrate);
```

Comment:
why we are hydrating?

Reply:
So if you click on the app logo, it's reloading the page instead of an ajax call. to fix this we need to hydrate this